smithy-swift-codegen/src/main/kotlin/software/amazon/smithy/swift/codegen/integration/HTTPBindingProtocolGenerator.kt

```diff
@@ -406,7 +406,7 @@ abstract class HTTPBindingProtocolGenerator(
         }
     }
 
-    override fun generateSmokeTests(ctx: ProtocolGenerator.GenerationContext) = SmokeTestGenerator(ctx).generateSmokeTests()
+    override fun generateSmokeTests(ctx: ProtocolGenerator.GenerationContext) = customizations.smokeTestGenerator(ctx).generateSmokeTests()
```

Comment:
Use the customizer's smoke test generator rather than creating a hard-coded generator